chat_shell/chat_shell/tools/events.py

```
@@ -52,9 +54,20 @@ def handle_tool_event(kind: str, event_data: dict):
     return handle_tool_event
 
 
+def _run_async(coro):
+    """Run async coroutine from sync context."""
+    try:
+        loop = asyncio.get_running_loop()
+        # We're in an async context, create a task
+        loop.create_task(coro)
```

Ruff (0.15.0)

[warning] Store a reference to the return value of `loop.create_task`

(RUF006)